uiowa/uiowa
# [Proof of concept] hiding header and footer

Resolves https://github.com/uiowa/uiowa/issues/2665. 

Like what we have set up with non-branded sites, I think this option would be useful for admins to use for customer requests like:

- Custom landing pages for social campaigns
- Custom one-off pages like admissions kiosk and slideshow
- Recreating a viewbook site
  - We might want to set more granular options like:
    - Hide main nav (useful for viewbooks)
    - Hide search
    - Hide header
    - Hide footer

# How to test
- Edit a page and click on the "Hide header and footer" checkbox
- Save page and verify that header and footer are not rendered in markup
